sdk/agentserver/azure-ai-agentserver-core/azure/ai/agentserver/core/client/tools/operations/_operations.py

```diff
@@ -0,0 +1,543 @@
+# ---------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# ---------------------------------------------------------
+
+import json
+import logging
+from typing import Any, Dict, List, Mapping, MutableMapping, Optional, Tuple, Union
+from azure.core import PipelineClient
+from .._configuration import AzureAIToolClientConfiguration
+from .._model_base import FoundryTool, ToolSource, UserInfo
+
+from .._utils._model_base import ToolsResponse, ToolDescriptorBuilder, ToolConfigurationParser, ResolveToolsRequest
+from .._utils._model_base import to_remote_server, MCPToolsListResponse, MetadataMapper
+from azure.core.pipeline import PipelineResponse
+from azure.core.rest import HttpRequest, HttpResponse
+
+from .._exceptions import OAuthConsentRequiredError
+
+from azure.core.exceptions import (
+    ClientAuthenticationError,
+    HttpResponseError,
+    ResourceExistsError,
+    ResourceNotFoundError,
+    ResourceNotModifiedError,
+    map_error,
+)
+
+logger = logging.getLogger(__name__)
+
+# Shared constants
+API_VERSION = "2025-05-15-preview"
```

Comment:
Are we not snapping to 2025-11-15-preview for all new ignite stuff that is in preview?